### PR TITLE
updated docs to correspond to the code

### PR DESCRIPTION
**Pull request by blublinsky**

### CODE REVIEW

File: data-processing-lib/doc/advanced-transform-tutorial.md [hunk 1]
```diff
@@ -38,9 +37,8 @@ found [here](../../transforms/universal/ededup/src/ededup_transform.py)
 
 First, let's define the transform class.  To do this we extend
 the base abstract/interface class
-[AbstractTableTransform](../src/data_processing/transform/table_transform.py),
+[AbstractTableTransform](../src/data_processing_ibm/transform/table_transform.py),
```
Comment (daw3rd) [inline data-processing-lib/doc/advanced-transform-tutorial.md]: This is the not in the library.

File: data-processing-lib/doc/advanced-transform-tutorial.md [hunk 2]
```diff
@@ -138,15 +136,14 @@ If there is no metadata then simply return an empty dictionary.
 
 First, let's define the transform runtime class.  To do this we extend
 the base abstract/interface class
-[DefaultTableTransformRuntime](../src/data_processing/ray/transform_runtime.py),
+[DefaultTableTransformRuntime](../src/data_processing_ibm/ray/transform_runtime.py),
```
Comment (daw3rd) [inline data-processing-lib/doc/advanced-transform-tutorial.md]: same

File: data-processing-lib/doc/advanced-transform-tutorial.md [hunk 3]
```diff
@@ -173,15 +170,15 @@ adds their handles to the transform parameters
         )
         return {"hashes": self.filters} | self.params
 ```
-Inputs to this method includes a set of parameters, that moght not be needed for this transformer, but 
+Inputs to this method includes a set of parameters, that moght not be needed for this transformer, but
 rather a superset of all parameters that can be used by different implementations of transform runtime (
-see for example [block listing](../../transforms/universal/blocklist), 
+see for example [block listing](../../transforms/universal/blocklisting),
```
Comment (daw3rd) [inline data-processing-lib/doc/advanced-transform-tutorial.md]: this is not in the open source (yet). And it is call `blocklist` not `blocklisting`.

File: data-processing-lib/doc/advanced-transform-tutorial.md [hunk 4]
```diff
@@ -217,9 +214,12 @@ define the following:
 First we define the class and its initializer,
 
 ```python
+short_name = "ededup"
+cli_prefix = f"{short_name}_"
+
 class EdedupTableTransformConfiguration(DefaultTableTransformConfiguration):
     def __init__(self):
-        super().__init__(name="ededup", runtime_class=EdedupRuntime, transform_class=EdedupTransform)
+        super().__init__(name=ededup, runtime_class=EdedupRuntime, transform_class=EdedupTransform)
```
Comment (daw3rd) [inline data-processing-lib/doc/advanced-transform-tutorial.md]: Where is ededup defined. Seems like we need `short_name` or "ededup"

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: data-processing-lib/doc/advanced-transform-tutorial.md (hunk 1 above)
Comment (blublinsky) [reply data-processing-lib/doc/advanced-transform-tutorial.md]: fixed

File: data-processing-lib/doc/advanced-transform-tutorial.md (hunk 2 above)
Comment (blublinsky) [reply data-processing-lib/doc/advanced-transform-tutorial.md]: fixed

File: data-processing-lib/doc/advanced-transform-tutorial.md (hunk 3 above)
Comment (blublinsky) [reply data-processing-lib/doc/advanced-transform-tutorial.md]: fixed

Comment (blublinsky) [reply data-processing-lib/doc/advanced-transform-tutorial.md]: fixed

File: data-processing-lib/doc/advanced-transform-tutorial.md (hunk 4 above)
Comment (blublinsky) [reply data-processing-lib/doc/advanced-transform-tutorial.md]: fixed

**Review by daw3rd — APPROVED** [review]

**Pull request merged**